src/main/resources/static/js/jquery.confirm.js

```diff
@@ -0,0 +1,147 @@
+/*!
+ * jquery.confirm
+ *
+ * @version 2.3.1
+ *
+ * @author My C-Labs
+ * @author Matthieu Napoli <matthieu@mnapoli.fr>
+ * @author Russel Vela
+ * @author Marcus Schwarz <msspamfang@gmx.de>
+ *
+ * @license MIT
+ * @url https://myclabs.github.io/jquery.confirm/
+ */
+(function ($) {
+
+    /**
+     * Confirm a link or a button
+     * @param [options] {{title, text, confirm, cancel, confirmButton, cancelButton, post, confirmButtonClass}}
+     */
+    $.fn.confirm = function (options) {
+        if (typeof options === 'undefined') {
+            options = {};
+        }
+
+        this.click(function (e) {
+            e.preventDefault();
+
+            var newOptions = $.extend({
+                button: $(this)
+            }, options);
+
+            $.confirm(newOptions, e);
+        });
+
+        return this;
+    };
+
+    /**
+     * Show a confirmation dialog
+     * @param [options] {{title, text, confirm, cancel, confirmButton, cancelButton, post, confirmButtonClass}}
+     * @param [e] {Event}
+     */
+    $.confirm = function (options, e) {
+        // Do nothing when active confirm modal.
+        if ($('.confirmation-modal').length > 0)
+            return;
+
+        // Parse options defined with "data-" attributes
+        var dataOptions = {};
+        if (options.button) {
+            var dataOptionsMapping = {
+                'title': 'title',
+                'text': 'text',
+                'confirm-button': 'confirmButton',
+                'cancel-button': 'cancelButton',
+                'confirm-button-class': 'confirmButtonClass',
+                'cancel-button-class': 'cancelButtonClass',
+                'dialog-class': 'dialogClass'
+            };
+            $.each(dataOptionsMapping, function(attributeName, optionName) {
+                var value = options.button.data(attributeName);
+                if (value) {
+                    dataOptions[optionName] = value;
+                }
+            });
+        }
+
+        // Default options
+        var settings = $.extend({}, $.confirm.options, {
+            confirm: function () {
+                var url = e && (('string' === typeof e && e) || (e.currentTarget && e.currentTarget.attributes['href'].value));
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=javascript%3AS1067)